openlibrary/plugins/openlibrary/home.py

```diff
@@ -38,9 +38,9 @@
 }
 
 
-def get_homepage():
+def get_homepage(devmode=False):
```

Comment:
One suggestion, let's not have a default here since we do expect it to always be specified :+1: This could mask errors happening in other places

```suggestion
def get_homepage(devmode):
```